CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
 ## Changelog
 
+### v1.4.6 Sep 18, 2024
```

Comment:
I think the latest entry in the changelog should be:

### v1.7.7 Oct 16, 2024
* Upgrade Reader SDK version support to 1.7.4 for iOS
* Support CompileSDK and targetSDK 34 on Android